src/Apps/Artwork/Components/ArtworkSidebar/index.tsx

```diff
@@ -1,7 +1,9 @@
-import { Separator, Serif } from "@artsy/palette"
+import { Separator, Serif, Spacer } from "@artsy/palette"
+import { Box } from "@artsy/palette"
+import { ContextConsumer } from "Artsy/Router"
+import Spinner from "Components/Spinner"
```

Comment:
import { Spinner } from '@artsy/palette'. (Not sure this has been updated in ArtistDetails, but should def come from palette.)